pkgs/servers/computing/slurm/default.nix

```diff
@@ -1,19 +1,24 @@
-{ stdenv, fetchurl, pkgconfig, curl, python, munge, perl, pam, openssl
+{ stdenv, fetchurl, pkgconfig, libtool, curl, python, munge, perl, pam, openssl
 , ncurses, mysql, gtk2, lua, hwloc, numactl
 }:
 
 stdenv.mkDerivation rec {
-  name = "slurm-llnl-${version}";
-  version = "15-08-5-1";
+  name = "slurm-${version}";
+  version = "17.02.6";
 
   src = fetchurl {
-    url = "https://github.com/SchedMD/slurm/archive/slurm-${version}.tar.gz";
-    sha256 = "05si1cn7zivggan25brsqfdw0ilvrlnhj96pwv16dh6vfkggzjr1";
+    url = "https://www.schedmd.com/downloads/latest/slurm-17.02.6.tar.bz2";
+    sha256 = "1sp4xg15jc569r6dh61svgk2fmy3ndcgr5358yryajslf1w14mzh";
   };
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ pkgconfig ];
+  # nixos test fails to start slurmd with 'undefined symbol: slurm_job_preempt_mode'
+  # https://groups.google.com/forum/#!topic/slurm-devel/HiltSkNiGJU
```

Comment:
Are you sure this was the correct link? The thread mentions something about removing "-O0" from `CFLAGS`.